framework/base/variables/_colors.scss

```diff
@@ -6,6 +6,7 @@ $ema-colors: (
     'blue-1': #13376d,
     'blue-2': #0b4ba0,
     'blue-3': #4172b3,
+    'blue-4': #0f3f82,
```

Comment:
Where do this color comes from?

Reply:
I colorpicked from the PDF. Should we use `primary`?